src/extensions/extra/brackets-cdn-suggestions

```diff
@@ -1 +1 @@
-Subproject commit f93ca8ccc5aa636e6a684a99cc50dea89240fb77
+Subproject commit f2b38ab78dc95e04a29819e9d094c77d55811a2f
```

Comment:
You need to rebase and/or `git submodule update` to get your tree to not have this change.